packages/pola-io/README.md

```diff
@@ -0,0 +1,119 @@
+# @agoric/pola-io
+
+A least-authority I/O library for JavaScript.
+
+## Getting Started
+
+```
+npm install @agoric/pola-io
+```
+
+**WIP WIP WIP - EXAMPLES BELOW ARE BROKEN**
+
+### Example: `agoric run` as `npx` subcommand
+
+```js
+import { makeCmdRunner } from '@agoric/pola-io';
+import { execFile as execFileNode } from 'node:child_process';
+import { promisify } from 'node:util';
+
+export const main = async ({ execFile = promisify(execFileNode) }) => {
+  const npx = makeCmdRunner('npx', { execFile }); // TODO --no-install
+  return miscTask(npx);
+};
+
+const miscTask = async npx => {
+  const agoric = npx.subCommand('agoric');
+  const { stdout } = await agoric.subCommand('run').exec(['builder.js']);
+  return stdout.split('\n');
+};
+```
+
+### Example: Read-Only File Wrapper
+
+```js
+import { makeReadOnlyFile } from '@agoric/pola-io';
+import * as fsp from 'node:fs/promises';
+import * as path from 'node:path';
+
+const file = {
+  async getBytes() {
+    return fsp.readFile('/etc/hosts', 'utf8');
+  },
+  async setBytes(_bytes) {
+    throw new Error('setBytes not permitted');
+  },
+};
+
+const readonly = makeReadOnlyFile(file);
+await readonly.getBytes(); // Allowed
+await readonly.setBytes(); // Throws or is not defined
+```
+
+## What This Package Offers
+
+The `@agoric/pola-io` package facilitates using the [Principle of Least Authority (POLA)](https://capnproto.org/capabilities.html#least-authority) for I/O, applying object-capability (ocap) principles to common file, network, and shell operations:
+
+- **Scoped Authority**: Interfaces are built to represent the smallest usable unit of authority—e.g., read-only file handles, command runners with fixed executable paths, or network dialers scoped to specific hosts.
+- **Composable Wrappers**: Inspired by Stiegler's PictureBook of Secure Cooperation, components like revocable forwarders, logging wrappers, and facets are encouraged as default patterns.
```

Comment:
I see an URL later on, but can you also make a link here for `Stiegler's _PictureBook of Secure Cooperation_`?

Reply:
Oops! I meant to take out that historical reference and leave that only in the Related Work section.